lib/util.js

```diff
@@ -4,7 +4,13 @@ function toOrderedArray(map) {
   return Object.keys(map).map(function(key) {
     return [key, map[key]];
   }).sort(function(a, b) {
-    return a[0] > b[0];
+    if (a[0] < b[0]) {
```

Comment:
this is a fix of the compare function, it should let `sort` function to work the same in any node version